perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -264,6 +265,11 @@ std::shared_ptr<Tracker> MultiObjectTracker::createNewTracker(
   const autoware_auto_perception_msgs::msg::DetectedObject & object,
   const rclcpp::Time & time) const
 {
+  if (use_pass_through_tracker_) {
+    std::cerr << "use pass through tracker" << std::endl;
+    return std::make_shared<PassThroughTracker>(time, object);
+  }
```

Comment:
I would like to define the mapping of labels to trackers as a yaml file, because readability will be improved.

Reply:
Since we would like to switch the tracker from the simlator.launch file, I guess it's easier to put the label in launch file.

Reply:
Readability is reduced by embedding special code for the simulator. :pray:

Reply:
Let's talk about this tomorrow

Reply:
done